src/ucs/time/time.h

```diff
@@ -72,6 +74,17 @@ static inline double ucs_time_sec_value()
     return ucs_get_cpu_clocks_per_sec();
 }
 
+/**
+ * @return The current time, without dependency on UCM code
```

Comment:
we want `dlopen(libucm.so)` to succeed without needing `libucs.so` (symbol: `ucs_arch_x86_enable_rdtsc`)

Reply:
maybe just call gettimeofday from UCM?

Reply:
fixed